.github/workflows/docker-test-flow.yml

```diff
@@ -139,7 +139,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: noir_wasm
-          path: /usr/src/noir/compiler/wasm/outputs/out/noir_wasm
+          path: /usr/src/noir/compiler/wasm
```

Comment:
Isn't this a breaking change for clients who download artifacts from here? Or there aren't any?

Reply:
This path is the path to be uploaded so there's no difference to consumers of this artifact.

Reply:
Yeah, we make no guarantees of stability for these artifacts. They're just a method of us parallelising github CI